### ISSUE TITLE
WCAG issue 4.1.2 on the combobox

### ISSUE THREAD

**Issue by olavflar** [Altinn/app-frontend-react#2517]
![Image](https://github.com/user-attachments/assets/1337c484-536b-45bf-a3d1-fd74631072b6)
### Description of the bug

WCAG-krav: 4.1.2
The value :35: is not allowed on the aria-controls attribute(s)
Gjelder denne kodesetningen:
<input aria-autocomplete=«llist»>

![Image](https://github.com/user-attachments/assets/2a685139-ab3c-47bf-a62f-2eddea6ebfa8)

### Steps To Reproduce

Test med en app - 
[https://udir.apps.tt02.altinn.no/udir/pbu-qa-anonym/](https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fudir.apps.tt02.altinn.no%2Fudir%2Fpbu-qa-anonym%2F&data=05%7C02%7Colav.friisberg.larssen%40digdir.no%7Caca4e70cf27c49c5c6e408dce158538a%7C008e560f08af4ceca056b35447503991%7C0%7C0%7C638633016683387825%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=ReRZA211ayh52DhnjywpTBl%2FpmNKrDEbDadnfwcxHTE%3D&reserved=0)

Oppdaterer med brukernavn og passord når jeg får det

### Additional Information

Design system issue: https://github.com/digdir/designsystemet/issues/2552

Her er hva du bør gjøre:

Finn eller opprett elementet med riktig id: Sørg for at det finnes et HTML-element på siden som har et id som matcher verdien du bruker i aria-controls. Eksempel:
html
Kopier kode
<input aria-autocomplete="list" role="combobox" aria-expanded="false" aria-controls="dropdown1" autocomplete="off" id="combobox-:r2fm:" class="fds-combobox__input" value="Norsk bokmål">
<ul id="dropdown1" class="dropdown-list">
  
</ul>
Erstatt verdien i aria-controls: Endre verdien til å matche ID-en til et eksisterende element. For eksempel, hvis du har en dropdown-liste med id="dropdown1", må aria-controls peke på dette elementet:
html
Kopier kode
<input aria-autocomplete="list" role="combobox" aria-expanded="false" aria-controls="dropdown1" autocomplete="off" id="combobox-:r2fm:" class="fds-combobox__input" value="Norsk bokmål">
WCAG 2.2 Success Criterion 4.1.2 handler om å sikre at navn, roller og verdier for elementer er tydelig definert for å sikre tilgjengelighet, spesielt for brukere som benytter hjelpemidler som skjermlesere. Å bruke en gyldig id i aria-controls er en del av denne prosessen.

For å rette opp feilen må du altså sørge for at ID-en som brukes i aria-controls faktisk eksisterer i HTML-dokumentet ditt.

_No response_

**Comment by Magnusrm** [comment]
The attribute that is causing the ARC toolkit test to fail is coming from the Designsystem. The wcag error is also present when testing with ARC toolkit on the design system page https://theme.designsystemet.no/ . Here the component is not rendered on a canvas like it is in the storybook, and we can then run tests on it with ARC toolkit.
